File: 프로그래머스/unrated/155652. 둘만의 암호/둘만의 암호.js
function solution(s, skip, index) {
  let canWords = Array.from({length: 26}, (value,index) => String.fromCodePoint(index+97)).filter(ele => !(skip.includes(ele)))

  let answer = ''
  for (let i=0; i<s.length; i++) {
    for (let j=0; j<canWords.length; j++) {
      if (s[i] === canWords[j]) {
        answer += canWords[(j+index)%(canWords.length)]
      }
    }
  }
  return answer;
}


console.log(solution("aukks", "wbqd",5))

File: 프로그래머스/1/155652. 둘만의 암호/둘만의 암호.py
def solution(s, skip, index):
    # a ~ z / 단, skip꺼 빼고
    bucket = [chr(x) for x in range(97,123) if chr(x) not in skip]
    print(bucket)

    result = ''
    for ele in s:
        print(bucket.index(ele)+5)
        result += bucket[(bucket.index(ele)+index) % (len(bucket))]

    return result

print(solution("aukks","wbqd",5)) # happy